rs/replicated_state/src/canister_state/system_state/wasm_chunk_store.rs

```diff
@@ -107,6 +107,16 @@ impl WasmChunkStore {
             })
     }
 
+    /// Prefer the paginating `get_chunk_data` unless you need the complete Chunk materialized.
```

Comment:
NIT: doc comment should explain what the function does and only then suggest what to do for other cases.

```
/// Returns the complete chunk data as a single vector.
/// 
/// Use `get_chunk_data` for paginated access.
```